docs/src/validator/best-practices/operations.md

```diff
@@ -0,0 +1,148 @@
+
+### Downloading Snapshots
+
+If you are starting a validator for the first time, or your validator has fallen too far behind after a restart, then you may have to download a snapshot.
+
+To download a snapshot, you must **_NOT_** use the `--no-snapshot-fetch` flag. Without the flag, your validator will automatically download a snapshot from your known validators that you specified with the `--known-validator` flag.
+
+If one of the known validators is downloading slowly, you can try adding the `--minimal-snapshot-download-speed` flag to your validator. This flag will switch to another known validator if the initial download speed is below the threshold that you set.
+
+### Manually Downloading Snapshots
+
+In the case that there are network troubles with one or more of your known validators, then you may have to manually download the snapshot. To manually download a snapshot from one of your known validators, first, find the IP address of the validator in using the `solana gossip` command. In the example below, `5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on` is the pubkey of one of my known validators:
+
+```
+solana gossip | grep 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on
+```
+
+The IP address of the validators is `139.178.68.207` and the open port on this validator is `80`. You can see the IP address and port in the fifth column in the gossip output:
+
+```
+139.178.68.207  | 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on | 8001   | 8004  | 139.178.68.207:80     | 1.10.27 | 1425680972
+```
+
+Now that the IP and port are known, you can download a full snapshot or an incremental snapshot:
+
+```
+wget --trust-server-names http://139.178.68.207:80/snapshot.tar.bz2
+wget --trust-server-names http://139.178.68.207:80/incremental-snapshot.tar.bz2
+```
+
+Now move those files into your snapshot directory. If you have not specified a snapshot directory, then you should put the files in your ledger directory.
+
+Once you have a local snapshot, you can restart your validator with the `--no-snapshot-fetch` flag.
+
+## Regularly Check Account Balances
+
+It is important that you do not accidentally run out of funds in your identity account, as your node will stop voting. It is also important to note that this account keypair is the most vulnerable of the three keypairs in a vote account because the keypair for the identity account is stored on your validator when running the `solana-validator` software. How much SOL you should store there is up to you. As a best practice, make sure to check the account regularly and refill or deduct from it as needed. To check the account balance do:
+
+```
+solana balance validator-keypair.json
+```
+
+> **Note** `solana-watchtower` can monitor for a minimum validator identity balance.  See [monitoring best practices](./monitoring.md) for details.
+
+## Regularly Withdraw From Vote Account
```

Comment:
Why "regularly" Arguably a vote account is more secure because it requires a special authority to withdraw from

Reply:
Fair. the regularity is up to the operator. I'll remove the "regularly" part. This section was more about details on withdrawing.